src/components/scenes/PasswordLoginScene.tsx

```diff
@@ -248,12 +248,11 @@ class PasswordLoginSceneComponent extends React.Component<Props, State> {
     const styles = getStyles(theme)
 
     return (
-      <FlatList
-        style={styles.dropDownList}
-        data={this.props.usernameOnlyList}
-        renderItem={this.renderRow}
-        keyExtractor={(item, index) => index.toString()}
-      />
+      <ScrollView style={styles.dropDownList}>
+        {this.props.usernameOnlyList.map(item => {
+          return <View key={item}>{this.renderRow({ item })}</View>
```

Comment:
Please get rid of the `renderRow` helper, and just return `<UserListItem  />` directly from the map. You can also put the `key` directly on this element, instead of the adding a superfluous view. Views have to cross the bridge into native code, so they are pricey.